components/vector-kubearchive-log-collector/development/loki-helm-dev-values.yaml

```diff
@@ -0,0 +1,189 @@
+    max_global_streams_per_user: 10000  # Reduce from 50000
+  ingester:
+    chunk_encoding: snappy
+    # Optimize for faster startup
+    chunk_target_size: 1048576  # 1MB chunks for faster processing
+    chunk_idle_period: 30s      # Shorter idle period
+    max_chunk_age: 10m          # Shorter max age for development
+  querier:
+    # Reduce concurrency for development
+    max_concurrent: 2  # Reduce from 4
+  pattern_ingester:
+    enabled: false
+
+# Distributed components configuration
+ingester:
+  replicas: 1  # Reduce from 3 to 1 for development
+  autoscaling:
+    enabled: false  # Disable autoscaling for dev
+  zoneAwareReplication:
+    enabled: false
+  resources:
+    requests:
+      cpu: 50m  # Reduce from 100m
+      memory: 128Mi  # Reduce from 256Mi
+    limits:
+      memory: 256Mi  # Reduce from 512Mi
+  persistence:
+    enabled: false  # Disable persistence for dev to avoid storage class issues
+    size: 10Gi
+  affinity: {}
+  podAntiAffinity:
+    soft: {}
+    hard: {}
+
+querier:
+  replicas: 1
+  autoscaling:
+    enabled: false
+  resources:
+    requests:
+      cpu: 50m  # Keep at 50m
+      memory: 128Mi  # Increase from 128Mi for safety
+    limits:
+      memory: 512Mi  # Restore to 512Mi as it needs more memory
+  affinity: {}
+
+queryFrontend:
+  replicas: 1
+  resources:
+    requests:
+      cpu: 25m  # Keep CPU low
+      memory: 128Mi  # Increase from 64Mi
+    limits:
+      memory: 256Mi  # Increase from 128Mi to 256Mi
+
+queryScheduler:
+  replicas: 1
+  resources:
+    requests:
+      cpu: 25m  # Reduce from 50m
+      memory: 32Mi  # Reduce from 64Mi
+    limits:
+      memory: 64Mi  # Reduce from 128Mi
+
+distributor:
+  replicas: 1  # Reduce from 3 to 1 for development
+  autoscaling:
+    enabled: false  # Disable autoscaling for dev
+  maxUnavailable: 1
+  resources:
+    requests:
+      cpu: 50m  # Reduce from 100m
+      memory: 128Mi  # Reduce from 256Mi
+    limits:
+      memory: 256Mi  # Reduce from 512Mi
+  affinity: {}
+
+compactor:
+  replicas: 1
+  resources:
+    requests:
+      cpu: 50m  # Reduce from 100m
+      memory: 64Mi  # Reduce from 128Mi
+    limits:
+      memory: 128Mi  # Reduce from 256Mi
+
+indexGateway:
+  replicas: 1
+  maxUnavailable: 0
+  resources:
+    requests:
+      cpu: 50m  # Reduce from 100m
+      memory: 128Mi  # Reduce from 256Mi
+    limits:
+      memory: 256Mi  # Reduce from 512Mi
+  affinity: {}
+
+ruler:
+  replicas: 0  # Disable ruler - not essential for basic log functionality
+
+# Disable all cache components to avoid Pending issues in development
+chunksCache:
+  enabled: false
+  replicas: 0
+
+resultsCache:
+  enabled: false
+  replicas: 0
+
+memcached:
+  enabled: false
+
+memcachedResults:
+  enabled: false
+
+memcachedChunks:
+  enabled: false
+
+memcachedFrontend:
+  enabled: false
+
+memcachedIndexQueries:
+  enabled: false
+
+memcachedIndexWrites:
+  enabled: false
+
+memcachedExporter:
+  enabled: false
+
```

Comment:
not sure but maybe the yaml lint will say something about this few white lines hehehe